config/samples/resources/computeregionnetworkendpointgroup/private-service-connection-region-network-endpoint-group/compute_v1beta1_computesubnetwork.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     cnrm.cloud.google.com/project-id: ${PROJECT_ID?}
 spec:
   region: us-west3
-  ipCidrRange: 10.2.0.0/16
+  ipCidrRange: 10.180.0.0/10
```

Comment:
This is e.g. to avoid a conflict in our test accounts?  I think that's fine - the actual value is unimportant in our samples.

Reply:
Yes to avoid conflict. We use 10.2.0.0/16 in another test and another region.